### DISCUSSION TITLE
Rich text field in collection

### DISCUSSION THREAD

**Discussion by nithin-techforce** [directus/directus#3823] [Archive]
Is there any way to create rich text field in directus collection so that it will helpful to post text,images,screenshots  for example in git hub while posting a question we can write the question and can drag and drop the related images there itsef

**Comment by rijkvanzanten** [comment]
You can use the WYSIWYG or Markdown interfaces 👍🏻

**Reply by benhaynes** [reply]
Correct — just be aware that the HTML WYSIWYG is missing the image tool as of now.

**Reply by nithin-techforce** [reply]
thank you for your reply ,can you navigate me the process of where to find that markdown option.i am unable to find it

**Reply by benhaynes** [reply]
It's under the `text` type during field setup.

**Reply by yellowstripe-io** [reply]
I dont see any option for WYSIWYG or Markdown:
